src/main/release-scripts/changelog-and-release-file-handling.rb

```diff
@@ -0,0 +1,172 @@
+
+# https://docs.atlassian.com/jira/REST/latest/#d2e2450
+$jira_issues_url='https://hibernate.atlassian.net/rest/api/2/search/?jql=project%20%3D%20OGM%20AND%20fixVersion%20%3D%20#{release_version}%20ORDER%20BY%20issuetype%20ASC'
+
+Choice.options do
+  header 'Application options:'
+
+  separator 'Required:'
+
+  option :release_version, :required => true do
+    short '-r'
+    long '--release-version=<version>'
+    desc 'The OGM release version to process'
+  end
+
+  separator 'Optional:'
+
+  option :website_release, :required => false do
+    short '-w'
+    long '--website-release-file'
+    desc 'If specified the hibernate.org release file for the specified release will be created'
+  end
+
+  option :update_changelog, :required => false do
+    short '-c'
+    long '--update-change-log=<path to changelog>'
+    desc 'If specified the hibernate.org release file for the specified release will be created'
+  end
+
+  separator 'Common:'
+
+  option :help do
+    short '-h'
+    long '--help'
+    desc 'Show this message.'
+  end
+end
+
+def get_json(url)
+  uri = URI.parse(url)
+  http = Net::HTTP.new(uri.host, uri.port)
+  http.use_ssl = true
+  http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+  request = Net::HTTP::Get.new(uri.request_uri)
+  response = http.request(request).body
+  JSON.parse(response)
+end
+
+# We are dealing with something like this
+#
+# ...,
+# {
+#     "self": "https://hibernate.atlassian.net/rest/api/latest/version/18754",
+#     "id": "18754",
+#     "description": "Bugfixes for MongoDB, Neo4j and CouchDB backends",
+#     "name": "4.1.2.Final",
+#     "archived": false,
+#     "released": true,
+#     "releaseDate": "2015-02-27",
+#     "userReleaseDate": "27/Feb/2015",
+#     "projectId": 10160
+# },
+# ...
+def get_release_info(release_version)
+  jira_versions = get_json $jira_versions_url
+  jira_version = jira_versions.find { |version| version['name'] == release_version }
+  abort "ERROR: Version #{release_version} does not exist in JIRA" if jira_version.nil?
+  abort "ERROR: Version #{release_version} is not yet released in JIRA" if !jira_version['released']
+  jira_version
+end
+
+def create_website_release_file(jira_release_info)
+  version = jira_release_info["name"]
+  puts 'Run the following command in the hibernate.org repository to create the release announcement file:'
+  puts ''
+  puts '====='
+  puts 'cat <<EOF > _data/projects/ogm/releases/' + jira_release_info['name'] + '.yml'
+  puts 'version: ' + version
+  puts 'version_family: ' + version.split('.')[0] + '.' + version.split('.')[1]
+  puts 'date: ' + jira_release_info['releaseDate']
+  puts 'stable: ' + (version.include?('Final') ? 'true' : 'false')
+  puts 'announcement_url: <TBD>'
+  puts 'summary: ' + (jira_release_info['description'] or '<TBD>')
+  puts 'displayed: true'
+  puts 'EOF'
+  puts '====='
+end
+
+def create_changelog_update(release_version)
```

Comment:
A short description would be nice, specifically of the format of change log records.